packages/app/src/components/Header/index.tsx

```diff
@@ -51,7 +52,7 @@ class HeaderComponent extends React.Component<Props, HeaderComponentState> {
 
 
 const styles = StyleSheet.create({
-    container: { backgroundColor: '#035EC7', height: 60, padding: 15, alignItems: 'center', justifyContent: 'space-between', flexDirection: 'row', width: '100%' },
+    container: { backgroundColor: '#035EC7', height: 60, padding: 20, alignItems: 'center', justifyContent: 'space-between', flexDirection: 'row', width: '100%' },
```

Comment:
what do you think about increasing padding to 80 ?

Reply:
@petermsouzajr I do not think it is a good idea to add much padding because it may cause responsive issues

Reply:
@petermsouzajr I have also added safeAreaView that will handle this issue.